CHANGES.md

```diff
@@ -1,5 +1,34 @@
 # Changes
 
+## edge-21.9.1
+
+This release includes various improvements and feature additions across the policy
+feature i.e, New validating webhook for policy resources. This also includes changes
+in the proxy i.e, terminating TCP connections when a authorization is revoked, improvements
+in the proxy authorization metrics. In addition, proxy injector has also been updated
+to set the right `opaque-ports` annotation on services with default opaque ports.
+
+* Added a new validating admission controller to validate the policy resources
+* Updated the proxy-init to remove a rule which caused the packets from the proxy
+  with destination != 127.0.0.1 on localhost to be sent to the inbound proxy
+* Added new `LINKERD_DOCKER_REGISTRY` env variable to configure the docker
+  registry in the CLI
```

Comment:
```suggestion
```

I don't think this is something we really want to advertise as a feature. It's more of a hack for our CI.